### ISSUE TITLE
CarPlay SB crash

### ISSUE THREAD

**Issue by jffortier249** [pwn20wndstuff/Undecimus#142]
I don't know who to address this problem. 

Lots of crash and SafeMode when using CarPlay. 

The main problem is if the iphone is still connect to CarPlay, it will respring and will always stay on Safemode, need to restart with the cable disconnected. 

Sometime, crashreport point Activator or Anemone as the suspect, remove both, same result, but now no suspect!

Let me know if you'll look into this, I will send crash log. 

It was doing the same thing with Electra, this is why a switch to Unc0ver!

Thanks

**Comment by Samgisaninja** [comment]
Possible duplicate of #134

**Comment by jffortier249** [comment]
Well my iphone8+ respring randomly often but it's a total different thing from issue134 (that doesn't provide a lot of info), this is CarPlay related, and I add some info... not just hey my device respring...

**Comment by Samgisaninja** [comment]
the issue here is, I'm pretty sure no one on the unc0ver team has a CarPlay compatible car, and that's not exactly something you can just go out and buy (at least, not cheaply), so we dont really have any hope of replicating this.

My only suggestion is, every time you get into the car, run `killall -11 SpringBoard` to enter safe mode. If the issue doesn't happen in safe mode, please close this issue as it's tweak related and not something on our end. If it still happens in safe mode, leave it open, but again, there's not much any of us can do with it.

**Comment by 877cms** [comment]
> I don't know who to address this problem.
> 
> Lots of crash and SafeMode when using CarPlay.
> 
> The main problem is if the iphone is still connect to CarPlay, it will respring and will always stay on Safemode, need to restart with the cable disconnected.
> 
> Sometime, crashreport point Activator or Anemone as the suspect, remove both, same result, but now no suspect!
> 
> Let me know if you'll look into this, I will send crash log.
> 
> It was doing the same thing with Electra, this is why a switch to Unc0ver! 
> 
> Thanks


Same exact thing here on iPhone 8, 11.3.1, lastest 2.0.2 unc0ver.
But I am using **CarBridge** tweak, not sure if the OP is using that? @jffortier249 

I'll try the "killall -11 SpringBoard", but I assume that will disable all tweaks including Carbridge? So I'm unsure what is proves? 

Thanks for the jailbreak btw :)

**Comment by sbingner** [comment]
Putting springboard into safemode won’t disable any tweaks in daemons outside of springboard.

 You need to jailbreak with tweaks disabled to be sure it’s not tweak related.  Does this ever happen when NOT jailbroken?

Sam

On Dec 12, 2018, at 6:38 AM, jffortier249 <notifications@github.com<mailto:notifications@github.com>> wrote:


I also use CarBridge and used NGXPlay before, same issue.

I will try to deactivate all tweaks on iCleaner, because using on safemode, I think, won't prove anything.

And in safemode, it will spring-loop if connected to CarPlay... so part of the problem is still present in safemode.

I'll do some testing and let you know.

**Comment by Mike130784** [comment]
I’m not sure if I should reply on this 
But I also had problems with CarPlay. 
Just to give some info at the moment 

In jailbroken state no tweaks. By connecting phone to CarPlay via USB it makes the phone go to safe mode as soon as CarPlay logo comes. 

In non jailbroken state it works fine. 

In jailbroken state with (my legit purchased) CarBridge tweak. CarPlay worked fine. 

iPhone 6 Plus. 11.3.1 unc0ver version 2.02

Verstuurd vanaf mijn iPhone

> Op 12 dec. 2018 om 19:56 heeft Sam Bingner <notifications@github.com> het volgende geschreven:
> 
> Putting springboard into safemode won’t disable any tweaks in daemons outside of springboard. 
> 
> You need to jailbreak with tweaks disabled to be sure it’s not tweak related. Does this ever happen when NOT jailbroken? 
> 
> Sam 
>

**Comment by Cryptiiiic** [comment]
@jffortier249 @Mike130784 Export your diagnostics from the unc0ver settings tab. And post them here.

**Comment by jffortier249** [comment]
I will have more time during the holidays to answer that post log.

And I would like also to point out, after it crash into SafeMode, maybe it's because by Saurik new substrate, but it don't spring loop anymore, at least this week... but if in safe mode and still connected to the CarPlay, the time will be freeze. I was on my way to an appointment and I went... "wow I still have plenty of time!" just to realsize I was 10 mins late !! :)

**Comment by Cryptiiiic** [comment]
@jffortier249 The holidays came and went, where is your diagnostics :thinking:. Also, still waiting on yours too @Mike130784.

**Comment by Mike130784** [comment]
@jffortier249 I’m so. So sorry for this late reply. 
At the time I just wanted to say what I had to say 
As it was a strange but similar problem 
By now I’m on unc0ver 2.1.0  
and I have not yet  And can not test at this moment if the same problem still occurs 
Because my work car which has CarPlay is coming back Wednesday 
Wednesday I will test if the same still happens. And post diagnostics 
Thank you for your patience though 
I had already forgotten about this 
Again my apologies

**Comment by BiasShadow** [comment]
I will try and add this too, however I havent actually had an issue with CarPlay since my last restore and jailbreak. The issue seems to really be post CarBridge, but I can't confirm.

**Comment by 877cms** [comment]
Hmm my Carplay just crashed on the way home, when pressing "Find my iPhone", I had to unplug usb and reconnect and it started working again. Nothing in analytics that matches the date/time... I have Carbridge installed too.

**Comment by Cryptiiiic** [comment]
@877cms It crashed once randomly for me after enabling an app to be displayed. (SpringBoard crashed) but after that I had no other crashes. I'll ask lefty about it. From your diagnostics, I don't see anything that would cause issues.

**Comment by 877cms** [comment]
@Cryptiiiic Actually similar here, working solid for weeks. Yesterday I had gotten in the car, arranged some of the apps in a different order and that's when it crashed after aprox 15mins.

Thanks for checking the diagnostics too 👍

**Comment by Mike130784** [comment]
>>The holidays came and went, where is your diagnostics 🤔. Also, still waiting on yours Mike130784<<

Hey mate i tested today as promissed
The problem seemed as far as i can tell. Solved in the new uncover. (2.1.0)
_________________________________________________________________________________________
Disabled all tweaks in icleaner. 
Connected phone to carplay. 
No crash to Safe Mode,  Where it previously went to safemode by just connecting. (2.0.2)

Enabled. Carbridge tweak. 
Everything still worked fine. 

Enabled all tweaks. 
And still everything worked fine.
________________________________________________________________________________________________
Made a diagnostics just in case ,
[diagnostics.zip](https://github.com/pwn20wndstuff/Undecimus/files/2741243/diagnostics.zip)

**Comment by Cryptiiiic** [comment]
Most likely Substrate related if it crashes in the future, closing.
